environment/create_google_config.py

```diff
@@ -0,0 +1,308 @@
+
+	# Ensure that gcloud SDK is installed, which is necessary to run the rest of the script. 
+	sdk_install_check()
+
+	# Select Google project (new or existing)
+	which_google_project()
+
+	# Create config
+	create_config()
+
+# Ensure that the user has not run this script before, to avoid overwriting an existing configuration file
+def google_config_check():
+	
+	# Check for .google_cromwell.config in ~/
+	home = os.path.expanduser("~")
+	existance = os.path.exists(home + '/.google_cromwell.config')
+	
+	# If there is a configuration file, exit
+	if existance:
+		print "\nYou already have a Cromwell configuration file. If you would like to clear this setup\nand create a new file, remove (or rename) the hidden file ~/.google_cromwell.config\n"
+		sys.exit("Exiting.")
+
+# Ensure that gcloud SDK is installed, which is necessary to run the rest of the script. 
+def sdk_install_check():
+		
+	# If gcloud SDK is not installed, install it.
+	if os.system('gcloud version') is not 0:
+		
+		# Ask if user wants to install the SDK
+		installSdk = raw_input('\nStep (1): You do not have Google Cloud SDK installed, which you need to run this script.\nDo you want to install gcloud SDK? (yes or no)').lower()
+		
+		while not (installSdk.startswith("y") or installSdk.startswith("n")):
+			installSdk = raw_input('\nPlease answer yes or no: ').lower()
+			
+		# User chooses to install
+		if installSdk.startswith("y"):
+			os.system('curl https://sdk.cloud.google.com | bash')
+			shell = os.path.expanduser("$SHELL")
+			# Need to create new shell to start using gcloud
+			os.system('exec -l ' + shell)
+			os.system('gcloud init')
+			return
+		
+		# User chooses not to install SDK, exit the script because they can't continue.
+		elif installSdk.startswith("n"):
+			sys.exit("Exiting.")
+	
+	# Gcloud SDK is already installed, and user can continue with setup
+	else:
+		print "\nStep (1): You have Google Cloud SDK installed."
+		return
+
+# Which Google Project to use (new or existing)
+def which_google_project():
+
+	existing_project = raw_input('\nStep (2): Do you have an existing Google project where you want to run workflows? (yes or no) ').lower()
+
+	while not (existing_project.startswith("y") or existing_project.startswith("n")):	
+		existing_project = raw_input('\nPlease answer yes or no: ').lower()
+		
+	# User has existing project
+	if existing_project.startswith("y"):
+		global project_name
```

Comment:
global should be fine here, or just use the project_name that which_google_project returns and pass that to the next function instead